.azure-pipelines/all.yml

```diff
@@ -0,0 +1,47 @@
+trigger:
+  batch: false
+  branches:
+    include:
+    - master
+
+pr:
+  branches:
+    include:
+    - master
+
+jobs:
+- job: UnitTests
+  pool:
+    vmImage: "Ubuntu-16.04"
+  strategy:
+    matrix:
+      Py27:
+        PYTHON_VERSION: "2.7"
```

Comment:
Do we need all those versions?

Reply:
It's what travis tested, figured I'd leave it like that